packages/bitcoin-rpc-provider/lib/BitcoinRpcProvider.js

```diff
@@ -198,7 +211,7 @@ export default class BitcoinRpcProvider extends JsonRpcProvider {
 
   async getTransactionByHash (transactionHash) {
     const tx = await this.getRawTransactionByHash(transactionHash, true)
-    const value = tx.vout.reduce((p, n) => p + parseInt(n.valueSat), 0)
+    const value = tx.vout.reduce((p, n) => p.plus(BigNumber(n.value).times(1e8)), BigNumber(0))
```

Comment:
`valueSat` does not exist in bitcoind

Reply:
I believe `valueSat` is only available on bitcore. We are going to remove bitcore support as they are very outdated (not updated for 2 years) and does not even support segwit.